CircuitPython_Templates/asyncio/code.py

```diff
@@ -76,22 +76,21 @@ async def monitor_button(button, animation_controls):
             key_event = key.events.get()
             if key_event:
                 if key_event.pressed:
-                    animation_controls.reverse = True
-                    animation_controls.delay = 0.1
+                    controls.reverse = True
+                    controls.delay = 0.1
                 elif key_event.released:
-                    animation_controls.reverse = False
-                    animation_controls.delay = 0.5
-            # Let another task run.
+                    controls.reverse = False
+                    controls.delay = 0.5
             await asyncio.sleep(0)
 
 
 async def main():
     animation_controls = AnimationControls()
-    buttons_task = asyncio.create_task(monitor_button(button_pin, animation_controls))
+    button_task = asyncio.create_task(monitor_button(button_pin, animation_controls))
     animation_task = asyncio.create_task(rainbow_cycle(animation_controls))
     blink_task = asyncio.create_task(blink(animation_controls))
 
     # This will run forever, because no tasks ever finish.
-    await asyncio.gather(buttons_task, animation_task, blink_task)
+    await asyncio.gather(button_task, animation_task, blink_task)
```

Comment:
Still called `animation_controls` here, and the class is `AnimationControls`. Do you want to change both of those to just `controls`/`Controls`?

Reply:
@dhalbert Nope! I want the `AnimationControls` bit to be separate. Because it contains the bits to control the animation. So I made the change to the animation functions only to separate out that they are expecting to control them, and that thing will eventually be `AnimationControls`.